### ISSUE TITLE
Feature request: Post-specific "catchup since" or order by date

### ISSUE THREAD

**Issue by sarahscheffler** [edemaine/coauthor#404]
It would be handy if there were a way to view recent changes *to a specific post*.  There are several different ways this could happen:
1. Within the post, have a "catchup since" feature or a "live feed" feature like the global one
2. Within the post, have a "sort by date" feature that still displays everything and keeps the nesting intact, but puts posts with newer changes higher.
3. Add a "filter by post title" feature to either the "catchup since" or "live feed" buttons.

**Comment by edemaine** [comment]
Agreed, and nice specific ideas!  (Assuming by "post" you mean "thread".)  This is closely related to #12 which we were discussing recently.

Probably the simplest/soonest feature we could get would be to add both `root:...` and `since:...` filters to search (#29), which is roughly your Suggestion 3. The slider idea in #12 is potentially a cooler version though.  I'll think about the other ideas...
